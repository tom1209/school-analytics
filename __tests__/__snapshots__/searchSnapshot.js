import { render } from "@testing-library/react";
import Search from "../../pages/search";

it('renders search page unchanged', () => {
  const { container } = render(<Search />)
  expect(container).toMatchSnapshot()
})