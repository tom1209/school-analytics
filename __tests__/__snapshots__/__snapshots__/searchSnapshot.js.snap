// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders search page unchanged 1`] = `
<div>
  <div>
    <div
      style="position: fixed; z-index: 9999; top: 16px; left: 16px; right: 16px; bottom: 16px; pointer-events: none;"
    />
    <div
      class="searchContainer"
    >
      <form
        autocomplete="off"
        class="searchForm"
      >
        <div>
          <label
            class="searchLabel"
          >
            Search for a school
          </label>
          <input
            class="searchInput"
            id="search"
            value=""
          />
        </div>
        <button
          class="searchButton"
          id="searchButton"
          type="submit"
        >
          Search
        </button>
      </form>
    </div>
  </div>
</div>
`;
